doc/source/user_guide/10min.rst

```diff
@@ -21,6 +21,14 @@ Object creation
 
 See the :ref:`Intro to data structures section <dsintro>`.
 
+:class:`Series` is a one-dimensional labeled array capable of holding any
+data type (integers, strings, floating point numbers, Python objects, etc.).
+
+:class:`Dataframe` is a 2-dimensional labeled data structure with columns of potentially different
+types. You can think of it like a spreadsheet or SQL table, or a dict of Series objects. Like
```

Comment:
To keep this succinct, I would remove the detail regarding what a `DataFrame` can accept (`Like Series, DataFrame accepts ....`). This information should be demo'd below along in the hyperlink created by ``` :class:`Dataframe` ```